tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -79,11 +111,6 @@ def do_build(args: CensusBuildArgs, skip_completed_steps: bool = False) -> int:
             args.state.commit(args.working_dir / CENSUS_BUILD_STATE)
             logging.info(f"{step_n_of}: complete")
 
-        logging.info("Census build: completed")
```

Comment:
We do have the log above (step n of m completed). I think that's enough. This log has been removed since this new method also deals with partial steps and not full builds anymore.